driver-moab/main/seq_frac_mct.F90

```diff
@@ -960,26 +959,26 @@ subroutine seq_frac_set(infodata, ice, ocn, fractions_a, fractions_i, fractions_
        call seq_frac_check(fractions_i,'ice set')
 
        ! MOAB
-       allocate(tagValues(arrSize) )
-       allocate(tagValues2(arrSize) )
-       allocate(tagValues3(arrSize) )
+       if (first_time) then
+          allocate(tagValues(arrSize_i) )
+          allocate(tagValues2(arrSize_i) )
```

Comment:
Is the `save` attribute on these variables added to prevent the allocate/deallocate here? Then, when will we deallocate the data? We can allocate during first_time=true, but without a last_time=true, how to deallocate memory? This pattern is fragile.

Reply:
right.  They don't need to be deallocated until the run ends.